tests/smoke-tests/src/main/resources/servers/audit-logging2/log4j2.properties

```diff
@@ -40,7 +40,7 @@ logger.audit_base = INFO, audit_log_file
 logger.audit_base.name = org.apache.activemq.audit.base
 logger.audit_base.additivity = false
 
-logger.audit_resource = OFF, audit_log_file
+logger.audit_resource = INFO, audit_log_file
```

Comment:
Related to other comment, the resource logger is enabled in the AuditLoggerResourceTest test config already.